build.gradle

```diff
@@ -63,12 +83,7 @@ dependencies {
 	runtime "com.yammer.metrics:metrics-core:$metricsVersion"
 	runtime "com.yammer.metrics:metrics-annotation:$metricsVersion"
 
-	compile("org.apache.kafka:kafka_$scalaVersion:$kafkaVersion") {
-		exclude module: 'jms'
-		exclude module: 'jmxtools'
-		exclude module: 'jmxri'
-	}
-
+	compile "org.apache.kafka:kafka_$scalaVersion:$kafkaVersion"
```

Comment:
H-m-m. I try to understand how have we missed IO plugin here...
Thank you very much, @wilkinsona !

Would you mind explain the change to this dependency?
Aren't those transitive dependencies there at all for Kafka 0.8.2?